.github/recipe.yaml

```diff
@@ -14,6 +14,7 @@ plugins:
   sqflite: ["wearable-5.5", "tv-6.0"]
   tizen_app_manager: ["wearable-5.5", "tv-6.0"]
   tizen_audio_manager: ["wearable-5.5", "tv-6.0"]
+  tizen_package_manager: ["wearable-5.5", "tv-6.0"]
```

Comment:
The integration test CI is failing because the example app requires signing with a platform certificate. We may skip this test for now by:

```
# Requires signing with a platform certificate.
tizen_package_manager: []
```

Reply:
I will remove install/uninstall package and package event test case because they need platform certificate.